checker/manual/constant-value-checker.tex

```diff
@@ -49,11 +49,31 @@ \subsection{Type Annotations\label{constant-value-checker-type-annotations}}
 The set is limited to 10 entries; if a variable
 could be more than 10 different values, the Constant Value
 Checker gives up and its type becomes
-\refqualclass{common/value/qual}{UnknownVal} instead.
+\refqualclass{common/value/qual}{UnknownVal} instead (except for
+\refqualclass{common/value/qual}{IntVal}
+, see below)
```

Comment:
Extra space, missing period.
Be specific about 
Improve writing to clarify distinction between integers and other types.